docs/configuration.md

```diff
@@ -532,6 +532,12 @@ It uses configuration from `exchange.pair_whitelist` and `exchange.pair_blacklis
 
 `refresh_period` allows setting the period (in seconds), at which the pairlist will be refreshed. Defaults to 1800s (30 minutes).
 
+`VolumePairList` is based on the ticker data, as reported by the ccxt library:
+
+* The `bidVolume` is the volume (amount) of current best bid in the orderbook.
+* The `askVolume` is the volume (amount) of current best ask in the orderbook.
+* The `quoteVolume` is the amount of quote (stake) currency traded (bought or sold) in last 24 hours.
```

Comment:
interrestingly, there would be `baseVolume` too - but we've excluded it (it's not in `SORT_VALUES`) - however i'm not certain why as there's no functional reason for that (probably it was missed ... ? )

Reply:
I guess, because baseVolume is in base currency, and the numbers differ in scale for the same stake currency (although ccxt documentation says that all amounts here in the quote currency)

Reply:
not for baseCurrency - you're right ... that's the reasion :+1: